crates/orderbook/src/api.rs

```diff
@@ -347,6 +348,7 @@ pub fn finalize_router(
         .recover(handle_rejection)
         .with(cors)
         .with(warp::log::log(log_prefix))
+        .with(warp::trace::trace(warp_tracing))
```

Comment:
Do we now add 2 info spans for the `request_id` (on in this layer and one in the `make_service_with_request_tracing` macro)?